src/routes/authentication-route.js

```diff
@@ -14,8 +15,18 @@ module.exports = function createAuthenticationRoute(repository) {
     return response.status(200).json({ token });
   }
 
+  async function comparePassword(request, response, next) {
+    const passwordDb = await repository.returnUser(request.body);
+    hashPassword.compare(request.body.password, passwordDb.user.dataValues.password)
+      .then((result) => {
+        if (result) return next();
+        return response.sendStatus(404);
+      })
+      .catch(console.error);
```

Comment:
for an exception in `compare` function, it would be good to inform the client not only print the error :)